admin/src/js/controllers/edit-user.js

```diff
@@ -291,6 +291,30 @@ angular
       return Object.keys(updates).length;
     };
 
+    var validateEmailAddress = function(){
+      if (!$scope.editUserModel.email){
+        return true;
+      }
+
+      if (!isEmailValid($scope.editUserModel.email)){
+        $translate('email.invalid').then(function(value) {
+          $scope.errors.email = value;
+        });
+        return false;
+      }
+
+      return true;
+    };
+
+    var isEmailValid = function(email){
+      if (email.match(/([\w-]+(?:\.[\w-]+)*)@((?:[\w-]+\.)*\w[\w-]{0,66})\.([a-z]{2,6}(?:\.[a-z]{2})?)\S+/)){
```

Comment:
(hi I'm not a reviewer I just saw this go over my feed)

Could this regex be replaced with `/.+@.+\..+/`, or something similarly simple?

Email validation is notoriously complicated, and the only real way to validate email addresses is to ask the domain.

For example, this regex fails on email addresses I've had in the past that are 100% valid. I'm not going to tell you which ones and why, because I don't want us to just fix this one case, as then we'll just fail for someone else ;-)